HomeWork/Program.cs

```diff
@@ -6,7 +6,19 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int[] Array = new int[10];
+            for (int index = 0; index < Array.Length; index++)
+            {
+                Array[index] = index * index;
+            }
+
+            foreach (int item in Array)
```

Comment:
You can output info in the first cycle. It's not necessary to create special cycle for output info.